e2e_tests/pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[build-system]
```

Comment:
You already mentioned this, but a README for how to use this would be helpful. Especially focusing on how to support a new plugin.